zeeguu/core/audio_lessons/prompts/meaning_lesson--teacher_challenges_both_dialogue_and_beyond.txt

```diff
@@ -30,10 +30,19 @@ Use as many challenges as possible from the dialogue.
 Add 4 more similar challenges from the teacher, but this time with examples out of the dialogue,
 with commonly used contexts for the word.
 
+In these challenges or any instruction from the teacher, make sure if the target word aka {origin_word} in the
```

Comment:
i think `i.e.` is more precise than `aka`

Reply:
Done, Is "resolve conversation" similar to "resolved issue"? I suppose